src/app/static/src/app/store/adr/actions.ts

```diff
@@ -140,6 +145,36 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
                     }
                 });
         }
+    },
+
+    async uploadFilestoADR(context, uploadFilesPayload) {
```

Comment:
How does this `uploadFilesPayload` argument relate to `uploadFiles` in the store? Is it the same thing?

Reply:
I wasn't sure you could use an array as a payload. I've now just simplified it to an array of UploadFile